src/editor.js

```diff
@@ -174,6 +174,11 @@ var bpIcons = {
  */
 var blocklyWorkSpace;
 
+/**
+ * The prop terminal object
+ */
+var pTerm;
```

Comment:
This is a global variable, so it should be placed in the globals.js file. I'll add a todo for that.